src/conditional-flow/numeric-conditions.js

```diff
@@ -2,21 +2,26 @@
 
 // This function should return true if there are no elements in the array, false otherwise
 function isArrayEmpty (array) {
-
+      return array.length === 0 ? true : false;
   // TODO: write code in this function body to pass the tests
 
 }
 
 // This function should return true if num1 is greater than num2, false otherwise
 function isGreaterThan (num1, num2) {
-
+      return num1 > num2 == true;
```

Comment:
You don't need the `== true` here. `num > num2` will create a boolean value. The computer will calculate this like so: 
```js
return num1 > num2 == true
return true == true
// or (depending on the values of num1/num2)
return false == true
```

So you could simplify it to:
```js
return num1 > num2
```